packages/admin/cms-admin/src/userPermissions/user/UserPageToolbar.tsx

```diff
@@ -38,7 +44,25 @@ export const UserPermissionsUserPageToolbar = ({ userId }: { userId: string }) =
             </ToolbarTitleItem>
             <ToolbarFillSpace />
             <ToolbarActions>
-                <StartImpersonationButton userId={userId} />
+                <CrudMoreActionsMenu
+                    overallActions={[
+                        isAllowed("impersonation")
+                            ? {
+                                  label: <FormattedMessage id="comet.impersonate" defaultMessage="Impersonate" />,
+                                  icon: <ImpersonateUser />,
+                                  disabled: userId === currentUser.id,
+                                  onClick: () => startImpersonation(userId),
+                              }
+                            : null,
+                        currentUser.impersonated
+                            ? {
+                                  icon: <Reset />,
+                                  label: <FormattedMessage id="comet.impersonate.stop" defaultMessage="Impersonate" />,
```

Comment:
Should the text here be different, as it has a different ID than the same text above?

Reply:
efd661ea3813416bdeddd8f44be5da33b38afc34